aws.go

```diff
@@ -6,17 +6,58 @@ import (
 	"time"
 
 	"github.com/aws/aws-sdk-go/aws"
-	"github.com/aws/aws-sdk-go/aws/session"
 	"github.com/aws/aws-sdk-go/service/cloudwatch"
+	"github.com/percona/rds_exporter/config"
 	"github.com/prometheus/client_golang/prometheus"
```

Comment:
Let's group imports the same way you do in `main.go`:
```go
import (
  "stdlib"

  "some/percona/lib"
  "third/party/lib"

  "imports/from/this/repo"
)
```

Reply:
> If this is now Percona standard for go libraries then please refer here to a doc mentioning it.

Proposed [here](https://confluence.percona.com/display/PMM/Common+tech%2C+libraries+and+approaches#Commontech,librariesandapproaches-Codestyle). I assumed you already follow the same practice by looking into `main.go` changes, but it sounds like it was accidental.

> Should I also modify all other our forked software to support the same convention?

If that proposal is accepted, then I would like us to follow the boy scout rule for small changes like this one. In other words, no need to fix it all at once, but fix it when you pass by.
* http://programmer.97things.oreilly.com/wiki/index.php/The_Boy_Scout_Rule
* https://martinfowler.com/bliki/OpportunisticRefactoring.html (this one also discusses dangers of bigger refactoring in the feature branches)